src/index.jsx

```diff
@@ -5,12 +5,27 @@ import { createStore, applyMiddleware } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
 import 'semantic-ui-css/semantic.min.css';
 import './index.scss';
-import App from './App';
+import 'react-redux-toastr/lib/css/react-redux-toastr.min.css';
+import thunk from 'redux-thunk';
+import ReduxToastr from 'react-redux-toastr';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Learn how importing works. We normally start with core modules at the top before importing our local files.